include.js

```diff
@@ -1261,6 +1261,14 @@ let cards = [
     author: 'Divyansh',
     githubLink: 'https://github.com/okaydivyansh'
   },
+
+  {
+    artName: 'wheel', // change this to the name of your artwork
+    pageLink: './Art/mohdUsman-6508/triangle.html', // change this
+    imageLink: './Art/mohdUsman-6508/wheel.gif', // change this
+    author: 'Mohd Usman', // use your name
+    githubLink: 'https://github.com/mohdUsman-6508' // change this
```

Comment:
Can you please remove all the comments from your entries? Thank you.

Reply:
i have removed all the comments.